client/src/components/container/admin/controlPage/component/ViewEvents/ViewEvents.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import { Link } from "react-router-dom";
+import { Table, message, Spin, Button } from 'antd';
+import axios from 'axios';
+
+import './ViewEvents.css'
+
+class ViewEvents extends Component {
+  state = {
+    error: null,
+    isLoaded: true,
+    events: [],
+    serverError: null,
+  }
+  
+  componentDidMount() {
+    axios.get('/api/v1/event')
+    .then(({ data }) => {
+      this.setState({
+        isLoaded: false,
+        events: data,
+      });
+    }).catch(() => {
+      this.setState({ serverError: "Internal server error !!", isLoaded: false })
+    })  }
+
+  handleAction = currentEvent => {
+    axios.patch("/api/v1/event", { id: currentEvent })
+      .then(({ data }) => {
+        const {data : { rows } } = data
+        message.success(data.msg, 10);
+        const remainingEvents = this.state.events.filter(event => event.id !== rows[0].id)
```

Comment:
use destructuring here: 
```suggestion
        const {events} = this.state;
        const remainingEvents = events.filter(event => event.id !== rows[0].id)
```